src/RestApi/StoreApi/Controllers/Cart.php

```diff
@@ -103,6 +103,44 @@ public function register_routes() {
 				'schema' => [ $this, 'get_public_item_schema' ],
 			]
 		);
+		register_rest_route(
+			$this->namespace,
+			'/' . $this->rest_base . '/remove-item',
+			[
+				[
+					'methods'  => 'POST',
+					'callback' => [ $this, 'remove_cart_item' ],
+					'args'     => [
+						'key' => [
+							'description' => __( 'Unique identifier (key) for the cart item.', 'woo-gutenberg-products-block' ),
+							'type'        => 'string',
+						],
+					],
+				],
+				'schema' => [ $this, 'get_public_item_schema' ],
+			]
+		);
+		register_rest_route(
+			$this->namespace,
+			'/' . $this->rest_base . '/update-item',
+			[
+				[
+					'methods'  => RestServer::EDITABLE,
```

Comment:
This route supports `EDITABLE` (i.e. `POST | PUT | PATCH`), consistent with the previous API from `cart/items`. @mikejolley I'm thinking we could use `EDITABLE` for all these cart modifier routes - thoughts? Alternatively could make them all `POST` only for consistency.

Reply:
Update - this is now `POST` for consistency with other `/cart` routes.